agents-docs/src/app/[[...slug]]/page.tsx

```diff
@@ -31,34 +26,17 @@ export default async function Page(props: PageProps<'/[[...slug]]'>) {
         style: 'clerk',
         enabled: tocEnabled,
       }}
-      breadcrumb={{
-        component: <Breadcrumb tree={source.pageTree} />,
-      }}
-      footer={{
-        component: <Footer url={page.url} />,
-      }}
-      container={{
-        className: 'lg:pt-0! [&>#nd-toc]:!pt-6 [&>#nd-toc]:pb-4 h-full min-h-0',
-        style: tocEnabled ? undefined : { '--fd-toc-width': 0 },
-      }}
     >
       <div className="flex items-center justify-between">
         <DocsTitle className="tracking-tight">{page.data.title}</DocsTitle>
         <PageControls
           title={page.data.title}
-          description={page.data.description ?? ''}
+          description={page.data.description}
           data={page.data.structuredData}
         />
       </div>
       {page.data.description && (
-        <div>
-          <Markdown
-            text={page.data.description}
-            components={{
-              p: (props) => <p {...props} className="text-lg text-fd-muted-foreground" />,
-            }}
-          />
-        </div>
```

Comment:
frontmatter description should **never** contain markdown, because it's injected in Next.js metadata description, so this compilation is unneeded